### PR TITLE
Tooltip: Fix wrapping for long text

### PR DESCRIPTION
**Pull request by francocorreasosa**
Continues #1258

* #1232 Add Tooltip maxWidth && fix some warnings about components without keys inside a map

* Tooltip max width is now fixed to 260px

### CODE REVIEW

**Comment by landitus** [comment]
FYI: This PR has an odd test because the tooltip gets behind the Storybook sidebar. I believe this is related to https://github.com/auth0/cosmos/issues/951. 

screen shot 2018-12-11 at 18 12 48

File: core/components/atoms/switch/switch.js [hunk 1]
```diff
@@ -29,12 +29,14 @@ class Switch extends React.Component {
   render() {
     let [onLabel, offLabel] = this.props.accessibleLabels
     let elements = [
-      <Checkbox type="checkbox" checked={this.state.on} readOnly id={this.props.id} />
+      <Checkbox key={0} type="checkbox" checked={this.state.on} readOnly id={this.props.id} />
```
Comment (siddharthkp) [inline core/components/atoms/switch/switch.js]: This is interesting, what does adding a hardcoded key do?

File: core/components/atoms/tooltip/tooltip.md [hunk 2]
```diff
@@ -9,23 +9,38 @@ description: Use tooltips for giving extra context AND to make visual cues acces
...
+
+### Tooltip Positions
 
 Tooltips can be attached to any of the four sides of an element using the `position` prop.
 
 ```js
 <Stack>
   <Tooltip position="top" content="Top">
-    <Icon name="copy" />
+    <Icon size={16} name="help" />
```
Comment (siddharthkp) [inline core/components/atoms/tooltip/tooltip.md]: at `size` prop 

Why is this needed in the example? 🤔

File: internal/docs/app.js [hunk 3]
```diff
@@ -106,8 +106,8 @@ class App extends React.Component {
           </SideContent>
           <MainContent id="main">
             <Body>
-              {guides.map(guide => (
-                <Route exact path={guide.path} component={guide.component} />
+              {guides.map((guide, i) => (
```
Comment (siddharthkp) [inline internal/docs/app.js]: thanks!

File: core/components/atoms/tooltip/tooltip.md (hunk 2 above)
Comment (landitus) [reply core/components/atoms/tooltip/tooltip.md]: Just to make the icon smaller and how a help tooltip would look like

Comment (landitus) [reply core/components/atoms/tooltip/tooltip.md]: No harm using a default.

File: core/components/atoms/switch/switch.js (hunk 1 above)
Comment (landitus) [reply core/components/atoms/switch/switch.js]: It’s there to silence warnings, as proposed by our contributor

Comment (siddharthkp) [reply core/components/atoms/switch/switch.js]: hard coded though?

File: core/components/atoms/tooltip/tooltip.md [hunk 4]
```diff
@@ -9,23 +9,38 @@ description: Use tooltips for giving extra context AND to make visual cues acces
...
+### Long text
+
+The Tooltip wraps long text by default to make it readable. Try to keep the content short and helpful for the user.
+
+```js
+<Tooltip
+  position="top"
+  content="This is a very long text. Please try to keep it no longer than 4 lines. The tooltip should be a short and helpful text."
+>
+  <Icon size={16} name="help" />
```
Comment (siddharthkp) [inline core/components/atoms/tooltip/tooltip.md]: Last one

```suggestion
  <Icon name="help" />
```

**Comment by siddharthkp** [comment]
Changed the description to be more release notes friendly 😄 

Tooltip: Fix wrapping for long text #1262
